omod/src/main/java/org/openmrs/module/owa/utils/OwaUtils.java

```diff
@@ -44,4 +44,12 @@ public static String removeVersionNumber(String passedFileName) {
 		}
 		return fileName;
 	}
+	
+	/**
+	 * @param requestURL URL to be checked
+	 * @return Boolean object to confirm whether URL contains "addonmanager"
+	 */
+	public static Boolean checkIfAddonManager(String requestURL) {
+		return requestURL.contains("addonmanager") ? true : false;
```

Comment:
Just in case there is a non owa module called addonmanage, how about checking for /owa/addonmanager/index.html

Reply:
Good idea